sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -135,6 +135,27 @@ ConfigurationAsyncClient client = new ConfigurationClientBuilder()
         .buildAsyncClient();
 ```
 
+You can also use `TokenCredential` to create a configuration client, such as AAD token. 
+If you using to AAD, make sure you have the endpoint of AppConfiguration service. You can find the end point from 
+connection string itself by looking for `endpoint={endpoint_value};`. You can get `TokenCredential` from other azure service
```

Comment:
I think we should give them the example of using the portal to find the AppConfiguration endpoint, if we say be that you can use a connection string to authenticate and say here use a connection string to find the endpoint I won't use AAD since I could just use a connection string and remove the hassle of using AAD.